tests/device_tests/test_msg_cardano_sign_transaction.py

```diff
@@ -17,215 +17,274 @@
 import pytest
 
 from trezorlib import cardano, messages
-from trezorlib.exceptions import TrezorFailure
-
-PROTOCOL_MAGICS = {"mainnet": 764824073, "testnet": 1097911063}
-
-SAMPLE_INPUTS = [
-    {
-        "input": {
-            "path": "m/44'/1815'/0'/0/1",
-            "prev_hash": "1af8fa0b754ff99253d983894e63a2b09cbb56c833ba18c3384210163f63dcfc",
-            "prev_index": 0,
-            "type": 0,
-        },
-        "prev_tx": "839f8200d818582482582008abb575fac4c39d5bf80683f7f0c37e48f4e3d96e37d1f6611919a7241b456600ff9f8282d818582183581cda4da43db3fca93695e71dab839e72271204d28b9d964d306b8800a8a0001a7a6916a51a00305becffa0",
-    }
-]
 
+NETWORK_IDS = {"mainnet": 0}
+
+SHELLEY_TEST_VECTORS_MNEMONIC = (
+    "test walk nut penalty hip pave soap entry language right filter choice"
+)
+
+# input flow sequence constants
+SWIPE = "SWIPE"
+YES = "YES"
+YIELD = "YIELD"
```

Comment:
This warrants a more detailed description. And perhaps grouping into a class with a descriptive yet short name? Do these constants correspond to some user actions? If yes, then UserAction would be a suitable name.